functions/source/soci-index-generator-lambda/Makefile

```diff
@@ -1,7 +1,7 @@
 default:
 	# Use static builds to make sure we don't have library version issues between the build env and lambda
-	GOOS=linux GOARCH=amd64 go build -tags "osusergo netgo static_build" -ldflags '-extldflags "-static"' -o main
-	zip soci_index_generator_lambda.zip main
+	GOOS=linux GOARCH=amd64 go build -tags "osusergo netgo static_build lambda.norpc" -ldflags '-extldflags "-static"' -o bootstrap
```

Comment:
This should still be called `main` since Handler is called main.

Reply:
Unless I'm understanding incorrectly I think we should probably change Handler to look for `bootstrap` instead of `main` (assuming such a change is needed, refer to below comment for elaboration).